### PR TITLE
Adding image_uri to docstring of runtime_env

### PR DESCRIPTION
**Pull request by japneet-anyscale**
Adding image_uri description and example to docstring. The parameter is in the signature, but not in the description section for the parameters.

cc: @jjyao

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by japneet-anyscale** [comment]
cc @angelinalg

**Comment by jjyao** [comment]
@japneet-anyscale lint failed, could you fix it?

File: python/ray/runtime_env/runtime_env.py [hunk 1]
```diff
@@ -266,6 +270,9 @@ class MyClass:
         config: config for runtime environment. Either
             a dict or a RuntimeEnvConfig. Field: (1) setup_timeout_seconds, the
             timeout of runtime environment creation,  timeout is in seconds.
+        image_uri: URI to a container image. The Ray worker process runs
+            in a container with this image. You can't use this parameter with
+            other fields of runtime_env.
```
Comment (zcin) [inline python/ray/runtime_env/runtime_env.py:275]: you can use it with `config` and `env_vars`

File: python/ray/runtime_env/runtime_env.py [hunk 2]
```diff
@@ -221,6 +221,10 @@ class MyClass:
         RuntimeEnv(
             pip={"packages":["tensorflow", "requests"], "pip_check": False,
             "pip_version": "==22.0.2;python_version=='3.8.11'"})
+        
+        # Example for using image_uri
+        RuntimeEnv(
+            image_uri="anyscale/ray:2.30.0-py39")
```
[the remark this answers is not in the document]
Comment (zcin) [reply python/ray/runtime_env/runtime_env.py:227]: +1

File: python/ray/runtime_env/runtime_env.py (hunk 1 above)
Comment (japneet-anyscale) [reply python/ray/runtime_env/runtime_env.py]: updated

**Comment by japneet-anyscale** [comment]
@jjyao @zcin I made the changes that you suggested

**Comment by japneet-anyscale** [comment]
@zcin @jjyao fixed image_uri wording and style per @angelinalg

**Review by zcin — APPROVED** [review]

**Review by angelinalg — APPROVED** [review]

**Pull request merged**